cs-amazon/src/main/java/io/cloudslang/content/amazon/utils/InputsUtil.java

```diff
@@ -368,6 +375,16 @@ public static void validateKeyOrValueString(String input, boolean isKey) {
         }
     }
 
+    public static String getValidArn(String input) {
+        Pattern pattern = Pattern.compile("[a-zA-Z0-9\\:\\-\\/]+");
```

Comment:
Isn't "\\\\:" a redundant escape? I think this can be reduced at this: "[a-zA-Z0-9:\\\\-/]+". Intellij should report this warning as well.